_data/tool_and_resource_list.yml

```diff
@@ -2660,3 +2660,48 @@
   id: better-bibtex
   name: Better BibTeX (BBT)
   url: https://retorque.re/zotero-better-bibtex/
+- description: GA4GH frames policy and builds standards to meet the real-world needs of the global genomics and health community.
+  id: ga4gh
+  name: Global Alliance for Genomics and Health (GA4GH)
+  url: https://www.ga4gh.org
+  registry:
+    biotools: ga4gh
+    fairsharing: 2tpx4v
+    tess: GA4GH
+- description: Discover tools and best practices for working with infectious disease data. IDTk provides general guidance as well as specific information for pathogen characterisation, socioeconomic data, human biomolecular data, and human clinical and health data.
+  id: idtk
+  name: Infectious Diseases Toolkit (IDTk)
+  url: https://www.infectious-diseases-toolkit.org/
+- description: The Beyond 1 Million Genomes (B1MG) project is helping to create a network of genetic and clinical data across Europe. The project provides coordination and support to the 1+ Million Genomes Initiative (1+MG). This initiative is a commitment of 24 EU countries, the UK and Norway to give cross-border access to one million sequenced genomes by 2022.
```

Comment:
I would say it is debatable to have projects such as "The Beyond 1 Million Genomes (B1MG)" as part of the table of resources. One would probably add specific outputs of the project, if relevant, not the project as a whole. This is my opinion, at least, but let us discuss, since we need to update the documentation for what goes in the table anyway. Any opinions, @bedroesb

Reply:
I agree, I took it over since it was described in the IDTk and I could easily take it over. But it's one of the examples we decided to be not part of the resources.

Reply:
Removed, resolving conversations.